### DISCUSSION TITLE
Crowdsec script ⚠️  Can't Install on Proxmox

### DISCUSSION THREAD

**Discussion by jubjubrsx** [community-scripts/ProxmoxVE#161] [Q&A] [answered]
just an fyi I tried to install crowdsec to check it out and I get the error
⚠️  Can't Install on Proxmox 
created a container to test and it just says error

**Comment by Mellowlynx** [comment, marked as answer]
It's a LXC add-on, it should be installed within a LXC, not the Proxmox host/node itself.
We will see if we can make this more clear on the frontend.

**Reply by jubjubrsx** [reply]
I created an ubuntu 24.04 container as well and just got some error? 
Does it need to be 23.04? 
That seems to be the case as I was playing with it the 24.04VM wouldnt install properly, but it would on 23.04.

**Reply by Mellowlynx** [reply]
Did you make a VM or your own LXC?
The LXC version on the site is still on 22.04
On a clean install, I have no problems running it.
![image](https://github.com/user-attachments/assets/48e30628-6be7-4046-8bdc-485aefd6728b)

**Reply by jubjubrsx** [reply]
Ok no worries I was using to high of a version it sounds like....
